model/trace/http.yaml

```
@@ -43,6 +43,8 @@ groups:
       - ref: url.full
         sampling_relevant: true
         requirement_level: required
+      - ref: http.route
```

Based on the discussion in the https://github.com/open-telemetry/semantic-conventions/issues/923, the suggestion is to reuse `url.template`

```suggestion
      - ref: url.template
```